docs/technical_reference/unit_models/translators/translator_adm1_asm2d.rst

```diff
@@ -69,15 +76,35 @@ Equations and Relationships
    "Volumetric flow equality", ":math:`F_{out} = F_{in}`"
    "Temperature balance", ":math:`T_{out} = T_{in}`"
    "Pressure balance", ":math:`P_{out} = P_{in}`"
-   "Fermentable substrate conversion", ":math:`S_{F, out} = Σ_{r1} C_{r1, in}`"
-   "Acetic acid conversion", ":math:`S_{A, out} = Σ_{r2} C_{r2, in}`"
-   "Unchanged component conversions", ":math:`C_{u, out} = C_{u, in}`"
-   "Ammonium conversion", ":math:`S_{NH4, out} = S_{IN, in}`"
-   "Phosphate conversion", ":math:`S_{PO4, out} = S_{IP, in}`"
-   "Biodegradable particulate organics conversion", ":math:`X_{S, out} = Σ_{s} C_{s, in}`"
    "Zero-flow component conversions", ":math:`C_{z, out} = 0`"
-
-
+   "Biomass concentration", ":math:`bio = X_{su, in} + X_{aa, in} + X_{fa, in} + X_{c4, in} + X_{pro, in} + X_{ac, in} + X_{h2, in} + X_{PAO, in}`"
+   "S_ac concentration", ":math:`S_{ac, 1} = S_{ac, in} + X_{PHA, in}`"
```

Comment:
Yes - the 1 and 2 subscripts correspond to the concentration of the given component at steps 1 and 2 `Sac_AD1.` This translator only has 2 steps (whereas the other translator has 5).

Reply:
A little bit confused here, e.g., I see you set `X_{PP,1} = 0` and then there are some conversion equations like `S_IP` will add `X_{PP,1}`, but it is set as 0, so what is the purpose for this conversion? Add 0 does not seem to have any impacts of the outcome. and where does `X_PP` convert to?

Reply:
I have done that just to be explicit, and the c-code also leaves in such terms even when they are set equal to 0. I wanted to avoid somebody cross-checking our code with the c-code and wondering why a term is missing from an expression

Reply:
X_PP is converted into S_Mg, S_K, and S_IP

Reply:
Yes, that is correct. It has no effect since it's zero. But is it possible that in the future we want to change X_PP from 0 to something else? If so, I think it's worth keeping this zero term here in the documentation and in the code itself. It's also just the most accurate representation of these equations.

Reply:
@adam-a-a hope you could also join the talk, I am confused because I believe X_PP should be important components of the system, right now, it seems no matter the initial value of X_PP is, it will directly set to 0 and no actual conversion happens to the system. Same conditions to X_PHA and X_PAO

Reply:
Oh... yea there's a mistake here. S_{IP, 1} should be X_{PP, in) + ... and not S_{IP, in) + ...

Reply:
Great catch @luohezhiming. This may or may not screw up the flowsheet again, but I'll test it out now...

Reply:
I will do a line-by-line review of the new translator file after this, which I just haven't had the time to do yet